src/decorators/extension.ts

```diff
@@ -0,0 +1,7 @@
+export function Extension(_name: string, _value: ExtensionType | ExtensionType[]): Function {
```

Comment:
What the reason to use underscore in variable names?

Reply:
To indicate that they're not used.